surveys/urls.py

```diff
@@ -6,12 +6,12 @@
 urlpatterns = [
     url(r'^forms/(?P<pk>\d+)/?$', views_api.SurveyView.as_view(), name='survey-form'),
     url(
-        r'^forms/(?P<pk>\d+)/respond/?$',
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>[^/]+)/?$',
         views_api.SurveyPostView.as_view(),
         name='survey-post',
     ),
     url(
-        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>.+)/?$',
+        r'^forms/(?P<pk>\d+)/respond/(?P<user_id>[^/]+)/latest/?$',
```

Comment:
Actually I have realised that we should make this a Get / Create endpoint as it seems overcomplicated to have to provide three urls to the frontend.